config/local.yml.dist

```diff
@@ -2,3 +2,4 @@ parameters:
     doctrine.website.algolia.admin_api_key: 'abcd'
     doctrine.website.projects_dir: '%doctrine.website.root_dir%/projects'
     doctrine.website.github.http_token: 'abcd'
+    doctrine.website.mysql.password: 'MyDBPassword'
```

Comment:
Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.